src/ripple/rpc/handlers/LedgerEntry.cpp

```diff
@@ -344,6 +345,168 @@ doLedgerEntry(RPC::JsonContext& context)
             jvResult[jss::error] = "malformedRequest";
         }
     }
+    else if (context.params.isMember(jss::bridge_account))
```

Comment:
The format of this RPC is a bit weird - it only asks for the door account. I would prefer to be able to pass in the whole XChainBridge object instead (perhaps with a key of `bridge`). One example situation for this is where I don't know which chain is which, but I have the XChainBridge contents (this is an actual issue with the Explorer).